src/Repository/ProfileField.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+namespace Friendica\Repository;
+
+use Friendica\BaseModel;
+use Friendica\BaseRepository;
+use Friendica\Collection;
+use Friendica\Database\Database;
+use Friendica\Database\DBA;
+use Friendica\DI;
+use Friendica\Model;
+use Friendica\Util\DateTimeFormat;
+use Psr\Log\LoggerInterface;
+
+class ProfileField extends BaseRepository
+{
+	protected static $table_name = 'profile_field';
+
+	protected static $model_class = Model\ProfileField::class;
+
+	protected static $collection_class = Collection\ProfileFields::class;
+
+	/** @var PermissionSet */
+	private $permissionSet;
+
+	public function __construct(Database $dba, LoggerInterface $logger, PermissionSet $permissionSet)
+	{
+		parent::__construct($dba, $logger);
+
+		$this->permissionSet = $permissionSet;
```

Comment:
this is the repository, isn't it? So `$this->repoPermissionSet` for more clear understanding

Reply:
Repositories don't hold any data/model anyway, I don't think it should be confusing.

Reply:
hm .. you're right, it's just if a dev dives into this class and didn't look right where she/he is to avoid the confusion.

Reply:
I expect the confusion to last exactly 10 seconds until they realize where they are and what they are doing.

Reply:
fair enough..